### DISCUSSION TITLE
Is there a function for calculating Path Loss in Sionna?

### DISCUSSION THREAD

**Discussion by alialsahlany** [NVlabs/sionna#550] [Q&A]
Hi,
Is there a specific function available in Sionna for calculating Path Loss?

**Comment by SebastianCa** [comment]
Hi @alialsahlany,
You can find an example of how to compute path loss in the [Diffraction Tutorial notebook](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Diffraction.html#Experiments-with-a-Simple-Wedge).

Since this isn’t directly related to an issue with Sionna, I’ll move this thread to the discussions section.

**Reply by alialsahlany** [reply]
Thank you for your suggestion. I want to look to control the transmitted power and measure the received power to calculate the Path Loss accurately. Any guidance on how to approach this within Sionna would be greatly appreciated.
